### PR TITLE
Add material themes Deep Ocean and Lighter

### PR DESCRIPTION
**Pull request by David-Aguilo**
Pull Request to add to NvChad the Material themes:

- Deep Ocean
- Lighter

Compared against the Material themes in JetBrains Products for C and Rust code using Clion and Rust-Rover respectively. Also used the material color palette: https://material-theme.com/docs/reference/color-palette/ and marco-cerovac's material nvim plugin: https://github.com/marko-cerovac/material.nvim as references.

Images comparing dmenu source code between nvim and clion:

- Deep Ocean

imatge

- Lighter

imatge

They aren't fully equal because things that appear the same under :Inspect have different colours in JetBrains. So I had to choose.

### CODE REVIEW

**Comment by siduck** [comment]
material-deep looks like pastel-dark.. I'll just add the light one

**Comment by siduck** [comment]
@David-Aguilo thanks for your PR,  i wanted to port this     theme by today soi commited from my side. Tons of changes were done!

**Comment by David-Aguilo** [comment]
> material-deep looks like pastel-dark.. I'll just add the light one

Hello. Thanks for the kind words. I have installed release 2.5 and checked pastelDark and feel it looks quite different to material-deep ocean:
imatge

Having pastelDark less contrast. I will continue to personally use material-deep ocean, though it is understandable if you feel it is unneeded.

**Comment by siduck** [comment]
@David-Aguilo in v3.0 we could add it then :) . Until then you use it as your local theme, and make sure the polish_hl has integration names -> hlgroups

for example u should wrap those hlgroups in treesitter table*
 
example : https://github.com/NvChad/base46/blob/v2.5/lua/base46/themes/solarized_osaka.lua#L59

**Comment by David-Aguilo** [comment]
Yes, was wondering why it was looking weird. Thank you for the explanation.

File: lua/base46/themes/material-deep-ocean.lua
Comment (s4nj1th) [inline lua/base46/themes/material-deep-ocean.lua:1]: used this. thank you.

**Comment by David-Aguilo** [comment]
@s4nj1th Glad you like it. It's been my daily driver since I made it. Hopefully it can be added to the repo eventually. Though I don't know if I should make a new Merge Request for it, given that this one is closed. Should I make a new one @siduck ?

**Comment by siduck** [comment]
sure @David-Aguilo

**Pull request closed without merging**